src/vm/eventpipeeventinstance.cpp

```diff
@@ -55,15 +55,17 @@ EventPipeEventInstance::EventPipeEventInstance(
     m_dataLength = length;
     QueryPerformanceCounter(&m_timeStamp);
     _ASSERTE(m_timeStamp.QuadPart > 0);
+#ifdef _DEBUG
+    EnsureConsistency();
+#endif // _DEBUG
+}
 
-    if(event.NeedStack() && !session.RundownEnabled())
+void EventPipeEventInstance::EnsureStack(EventPipeSession &session)
```

Comment:
Could you please make the argument ~`EventPipeSession &session`~ -> `const EventPipeSession &session`?